colin-api/src/colin_api/models/filing.py

```diff
@@ -1396,8 +1397,10 @@ def add_filing(cls, con, filing: Filing) -> int:
                                                 Business.TypeCodes.BCOMP_CONTINUE_IN.value,
                                             ])
 
-            # Freeze all entities except CP if 'enable-bc-ccc-ulc' flag is on else just freeze BEN
+            # Freeze all entities except CP if business exists in lear and
+            # 'enable-bc-ccc-ulc' flag is on else just freeze BEN
             is_frozen_condition = (
+                cls.is_business_in_lear(bc_identifier) and
```

Comment:
can we actually assign this to a variable and add two log statements?  

it would be nice to be able to tell what the value is for `is_business_in_lear` and `is_frozen_condition` from the logs.

Reply:
Actually, i'm contradicting myself as that would require doing the `is_business_in_lear` call upfront.

Maybe just log `is_frozen_condition`.

The http call in LEAR might be logged automatically anyways

Reply:
okay